data/mods/My_Sweet_Cataclysm/sweet_monster_groups.json

```diff
@@ -38,5 +38,10 @@
     "name": "GROUP_EGG_CHOC_COW",
     "type": "monstergroup",
     "default": "mon_cow_calf_choc"
+  },
+  {
+    "name": "GROUP_EGG_LICORICE",
+    "type": "monstergroup",
+    "default": "mon_licorice_snake"
```

Comment:
Maybe make a baby version of the snake that would spawn from the egg and then evovle into grown up snake

Reply:
Where do I put the baby version, In regular monsters.json?

Reply:
Yep same as the other monsters

Reply:
Ok, will do this today. I'll use chocolate calf as example of how this should work.